packages/puppeteer-core/src/api/Page.ts

```diff
@@ -318,11 +318,19 @@ export interface ScreenshotOptions {
  * @public
  * @experimental
  */
+export type FileFormat = 'gif' | 'webm' | 'mp4';
+
 export interface ScreencastOptions {
```

Comment:
There is a missing comment with TsDoc, please copy it to this interface from the FileFormat.